### PR TITLE
GB-24: Tests for frontend from milestone #1

### PR DESCRIPTION
**Pull request by ksadoff**
## Jira Ticket Number
https://gaming-backlog.atlassian.net/browse/GB-24

## Short description
Creates test files for all frontend pages created in Milestone 1 

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Documentation
- [x] Other (please specify): Testing

### CODE REVIEW

File: src/test/unit/components/GameCard.test.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        gameDescription='Surprisingly few dragons.'
+        gameImage=''
+     />);
+  });
+
+  it('renders GameCard', () => {
+    const text = screen.getByText(/Add to Library/i);
+    expect(text).toBeInTheDocument();
+  })
+  // TODO: Once functionality in GameCard is working, uncomment these tests
```
Comment (ksadoff) [inline src/test/unit/components/GameCard.test.tsx:21]: https://gaming-backlog.atlassian.net/browse/GB-26

File: src/test/unit/components/GameCard.test.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { GameCard } from '../../../components/GameCard';
+
+// const img = "path-to-image"
```
Comment (trauman413) [inline src/test/unit/components/GameCard.test.tsx:5]: do you need this comment?

File: src/test/unit/pages/GamePage.test.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { GamePage } from '../../../pages/GamePage';
```
Comment (trauman413) [inline src/test/unit/pages/GamePage.test.tsx:3]: Created ticket for making this absolute: https://gaming-backlog.atlassian.net/jira/software/projects/GB/boards/1?selectedIssue=GB-37

no need for work on your end for this, just making note of this as it's bothering me :)

**Review by trauman413 — APPROVED** [review]
small comments that don't need fixing, approved!

File: src/test/unit/pages/GamePage.test.tsx (hunk 3 above)
Comment (ksadoff) [reply src/test/unit/pages/GamePage.test.tsx:3]: Ah perfect, thanks!

File: src/test/unit/components/GameCard.test.tsx (hunk 2 above)
Comment (ksadoff) [reply src/test/unit/components/GameCard.test.tsx:5]: Not strictly but I figured I'd leave it in as a placeholder for when we actually implement it

**Pull request merged**